admin/view-translations-post.php

```diff
@@ -6,31 +6,32 @@
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Don't access directly
+	exit; // Don't access directly.
```

Comment:
As you took the opportunity to rewrite some other things, you can also rewrite this condition as we decided `defined( 'ABSPATH' ) || exit;`.